### PR TITLE
Moving config classes to own java package

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Simultaneously moving code currently under config to configgen.

### CODE REVIEW

File: src/main/java/com/google/api/codegen/config/BundlingConfig.java [hunk 1]
```diff
@@ -12,8 +12,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.api.codegen;
+package com.google.api.codegen.config;
 
+import com.google.api.codegen.ConfigProto;
```
Comment (michaelbausor) [inline src/main/java/com/google/api/codegen/config/BundlingConfig.java]: Should ConfigProto, BundlingConfigProto, etc. also be moved into codegen.config package?

Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/config/BundlingConfig.java]: I want to do that, but it's a breaking change for all GAPIC config out there, so I'd prefer to defer until another day.

Comment (michaelbausor) [inline src/main/java/com/google/api/codegen/config/BundlingConfig.java]: Fair enough, maybe create an issue for this?

**Comment by michaelbausor** [comment]
LGTM

File: src/main/java/com/google/api/codegen/config/BundlingConfig.java (hunk 1 above)
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/config/BundlingConfig.java]: https://github.com/googleapis/toolkit/issues/563

**Pull request merged**